Anscombe_script.py

```diff
@@ -0,0 +1,53 @@
+import seaborn as sns
+import os
+
+
+SAVE_DIR = os.path.join(os.getcwd(), "Anscombe's quartet")
+
+def anscombe_data():
+    """
+    This function loads the Anscombe's quartet from the Seaborn's library resources
+    and calculates mean, standard deviation and variance
+    Args:
+        None
+
+    Returns:
+          df - dataframe with Anscombe's x and y values
+          description - statistical description of Anscombe's datasets
+    """
+    #Loading data from seaborn resources
+    data = sns.load_dataset('anscombe')
+
+    #Creating data frame with calculated mean and standard deviation
+    description = data.groupby('dataset').describe().loc[:,(slice(None),['mean','std'])]
+
+    #Calculating variance and merging both dataframes
+    variance = data.groupby('dataset').var()
+    variance.rename(columns={'x': '(x, var)', 'y': '(y,var)'}, inplace=True)
+    description = description.join(variance).round(2)
```

Comment:
UserWarning: merging between different levels can give an unintended result (2 levels on the left,1 on the right)
  warnings.warn(msg, UserWarning)
Try to use `pd.concat([description, variance.round(2)], axis=1)` to avoid warning